cluster-up/cluster/kind-1.22-sriov/sriov-components/manifests/multus/kustomization.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- multus.yaml
+images:
+- name: nfvpe/multus:v3.4
+  newName: quay.io/kubevirtci/multus
+  newTag: v3.4
```

Can we keep the incoming tag so we just replace the name ? 
```suggestion
- name: nfvpe/multus
  newName: quay.io/kubevirtci/multus
```

---

Its possible, I specified the tag for visibility.
You prefer to remove it anyway?

---

Let's remove it, If we bump multus we will not need to update this file.

---

Done